Test.Microsoft.NET.Build.Containers.Filesystem/LayerEndToEnd.cs

```diff
@@ -21,19 +22,10 @@ public void SingleFileInFolder()
         Console.WriteLine(l.Descriptor);
 
         //Assert.AreEqual("application/vnd.oci.image.layer.v1.tar", l.Descriptor.MediaType); // TODO: configurability
-        Assert.AreEqual(2048, l.Descriptor.Size);
+        Assert.IsTrue(l.Descriptor.Size is >= 135 and <= 200, $"'l.Descriptor.Size' should be between 135 and 200, but is {l.Descriptor.Size}"); // TODO: determinism!
```

Comment:
Note - I was going to look into determinism at some point, but got pulled off for other things. I think the issue is that `TarWriter.WriteEntry(item.path, containerPath);` also captures file metadata (DateCreated, etc) in the resulting tar. I think we will need to create the entry manually in order to make it deterministic.

Reply:
We can leave that for dotnet/sdk-container-builds#34 then, IMO.